create-publication-from-scopus/src/test/resources/2-s2.0-0000469852.xml

```diff
@@ -176,16 +176,7 @@
                             </preferred-name>
                         </author>
                     </author-group>
-                    <abstracts>
-                        <abstract original="y" xml:lang="eng">
-                            <ce:para>The identifications of taxa of larger fungi (basidiomycetes) covering collections
-                                from the Korup National Park, Cameroon is given with relevant details on the
-                                distribution elsewhere in the world if known. A few other records from the south western
-                                province are also given. Several rare and previously unrecorded fungi for the area are
-                                documented. © 1993, Trustees of the Royal Botanic Garden Edinburgh. All rights reserved.
-                            </ce:para>
-                        </abstract>
-                    </abstracts>
+                    <abstracts><abstract original="y" xml:lang="eng"><ce:para>The identifications of taxa of larger fungi (basidiomycetes) covering collections from the Korup National Park, Cameroon is given with relevant details on the distribution elsewhere in the world if known. A few other records from the south western province are also given. Several rare and previously unrecorded fungi for the area are documented. © 1993, Trustees of the Royal Botanic Garden Edinburgh. All rights reserved.</ce:para></abstract></abstracts>
```

Comment:
Could not figure out why there was so much white space in the output abstract object. LOL